riva/clients/nmt/streaming_s2s_client.cc

```diff
@@ -318,6 +318,10 @@ StreamingS2SClient::ReceiveResponses(std::shared_ptr<S2SClientCall> call, bool a
   std::vector<int16_t> pcm_buffer;
   std::vector<unsigned char> opus_buffer;
   while (call->streamer->Read(&call->response)) {  // Returns false when no more to read.
+    if (call->response.has_is_final()) { // End of stream for TTS, exiting while.
+        std::cout<< "TTS end of stream :" << call->response.is_final() << std::endl;
```

Comment:
we can remove this log, end of stream should not be logged by default since this log is not meaningful for someone using the streaming mode.

Reply:
done.